File: seed/java-sdk/exhaustive/local-files/src/main/java/com/snippets/Example59.java
package com.snippets;

import com.fern.sdk.SeedApiClient;
import com.fern.sdk.resources.endpointsparams.requests.EndpointsParamsModifyWithPathRequest;

public class Example59 {
    public static void main(String[] args) {
        SeedApiClient client = SeedApiClient
            .builder()
            .token("<token>")
            .url("https://api.fern.com")
            .build();

        client.endpointsParams().endpointsParamsModifyWithPath(
            EndpointsParamsModifyWithPathRequest
                .builder()
                .param("param")
                .body("string")
                .build()
        );
    }
}

File: seed/csharp-sdk/exhaustive/use-undiscriminated-unions/src/SeedApi.DynamicSnippets/Example59.cs
using SeedApi;

namespace Usage;

public class Example59
{
    public async Task Do() {
        var client = new SeedApiClient(
            token: "<token>",
            clientOptions: new ClientOptions {
                BaseUrl = "https://api.fern.com"
            }
        );

        await client.EndpointsParams.EndpointsParamsModifyWithPathAsync(
            new EndpointsParamsModifyWithPathRequest {
                Param = "param",
                Body = "string"
            }
        );
    }

}
